service-front/module/Application/src/Model/Service/System/Status.php

```diff
@@ -71,93 +57,59 @@ public function check()
         return $result;
     }
 
-    //------------------------------------------------------------------------
-
     private function dynamo()
     {
-        $result = array('ok' => false, 'details' => [
-//            'sessions' => false,
-            'locks' => false,
-        ]);
-
-//        //------------------
-//        // Sessions
-//
-//        try {
-//            $details = $this->dynamoDbSessionClient->describeTable([
-//                'TableName' => $this->getConfig()['session']['dynamodb']['settings']['table_name']
-//            ]);
-//
-//            if ($details['@metadata']['statusCode'] === 200 && in_array($details['Table']['TableStatus'], ['ACTIVE', 'UPDATING'])) {
-//                // Table is okay
-//                $result['details']['sessions'] = true;
-//            }
-//        } catch (Exception $e) {}
-
-        //------------------
-        // Locks
+        $result = array(
+            'ok' => false,
+        );
 
+        // DynamoDb (system message table)
         try {
-            $details = $this->dynamoDbCronClient->describeTable([
-                'TableName' => $this->getConfig()['cron']['lock']['dynamodb']['settings']['table_name']
-            ]);
-
-            if ($details['@metadata']['statusCode'] === 200 && in_array($details['Table']['TableStatus'], ['ACTIVE', 'UPDATING'])) {
+            $details = $this->dynamoDbClient->describeTable([
```

Comment:
So some DynamoDb stuff is staying for now, looks like for the system message? Could the description of this PR perhaps be modded to point at the story for that?  Presumably getting rid of DynamoDb completely is a series of stories?  Would be good to be able to see the pathway..

Reply:
I am not worried about how we get rid of it altogether as far as this story goes. There is a ticket to move the system message out of dynamodb: https://opgtransform.atlassian.net/browse/LPAL-752. This was closed as WONTDO a few weeks back. We could re-open it if people think it's worthwhile.

Reply:
https://opgtransform.atlassian.net/browse/LPAL-744 is the epic we had to totally remove dynamodb. I've asked Sarah M. why we closed this as I can't recall. Probably because it's a lot of work for little benefit.

Reply:
Thanks for explanation :)